### PR TITLE
GEOMESA-1486 Fix Kafka links in installation docs.

### PR DESCRIPTION
**Pull request by mdzimmerman**
(After review I will cherry-pick to 1.3.0-m1 docs branch.)

Signed-off-by: Matthew Zimmerman <matt.zimmerman@ccri.com>

### CODE REVIEW

**Comment by aheyne** [comment]
Looks good to me.

File: docs/user/installation_and_configuration.rst [hunk 1]
```diff
@@ -651,13 +656,26 @@ Kafka you are running.
...
 
-* Kafka
+* Kafka 0.8
     * kafka-clients-0.8.2.1.jar
     * kafka_2.11-0.8.2.1.jar
     * metrics-core-2.2.0.jar
     * zkclient-0.3.jar
-* Zookeeper
-    * zookeeper-3.4.5.jar
+    * zookeeper-3.4.5.jar or zookeeper-3.4.6.jar
```
Comment (jahhulbert-ccri) [inline docs/user/installation_and_configuration.rst]: i'd probably just recommend 3.4.6 at this point

**Comment by jahhulbert-ccri** [comment]
looks good

**Comment by mdzimmerman** [comment]
Fixed. Thanks!

**Pull request merged**